templates/windows/chocolateyUninstall.ps1.erb

```diff
@@ -0,0 +1,20 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$contentFile=(join-path -path "$toolsDir" -childpath "file-list") + ".txt"
+if (Test-Path -path "$contentFile") {
+  Write-Debug "Removing files defiend in `'$contentFile`'"
+  $contentFile
+  $contents=get-content "$contentFile"
```

Comment:
might be good to know. But it does look like it at least pulls the first item out of there.